src/js/app.js

```diff
@@ -17,11 +17,15 @@ var $shareLink = document.getElementById('share-link')
 
 var componentCtors = require('./components')
 var components = [
-  componentCtors.Help('help'),
+  componentCtors.Header('header', {
+    create: initArchive,
+    download: initArchive,
+    title: 'datland'
+  }),
+  //componentCtors.Help('help'),
   componentCtors.HyperdriveSize('hyperdrive-size'),
   componentCtors.HyperdriveStats('hyperdrive-stats'),
   componentCtors.Peers('peers'),
-  componentCtors.ResetButton('new', initArchive),
   componentCtors.SpeedDisplay('speed')
 ]
```

Comment:
this is not a comment about this PR, but just a general note for future: 
im going to add an `action.js` file for calling methods on `store.js`, and a `connector.js` file for initializing components and connecting them to the mindux store so component initialization doesn't balloon uglily here in `app.js`.